src/store.js

```diff
@@ -1,14 +1,14 @@
 var throttle = require('./lib/throttle');
 var toDenseArray = require('./lib/to_dense_array');
-var SimpleSet = require('./lib/simple_set');
+var StringSet = require('./lib/string_set');
 var render = require('./render');
 
 var Store = module.exports = function(ctx) {
   this._features = {};
-  this._featureIds = new SimpleSet();
-  this._selectedFeatureIds = new SimpleSet();
-  this._changedFeatureIds = new SimpleSet();
-  this._deletedFeaturesToEmit = new SimpleSet();
+  this._featureIds = new StringSet();
+  this._selectedFeatureIds = new StringSet();
+  this._changedFeatureIds = [];
```

Comment:
I suspect that both ways are ok and that we wouldn't see much of a perf change because of the size. Might be good to keep this a Set just so all of the ids lists are sets.